### DISCUSSION TITLE
Run time ml.simulate

### DISCUSSION THREAD

**Discussion by onadevos** [phydrus/phydrus#63] [General]
Hi
I'm just getting started on phydrus. I'm trying to run exercise 1 and the input files are written just fine but Hydrus never finishes running. I don't receive any errors so I don't know what's wrong...

**Comment by martinvonk** [comment]
Could be something with your executable. Did you manage to compile it successfully?

**Reply by onadevos** [reply]
I had some trouble compiling it, but I was able to create an hydrus.exe using make.bat . I assumed it was created correctly once I got an executable. Is there a way to check this?

**Reply by martinvonk** [reply]
Unfortunately no, can you post your input files here? Then I can have a look.

**Reply by onadevos** [reply]
I was reviewing the input files to send them, when I realised the input file wasn't read out correctly. Now it works. Thank you!
